src/lib/hash.h

```diff
@@ -85,4 +86,16 @@ pgp_hash_alg_t pgp_hash_alg_type(const pgp_hash_t *hash);
 
 pgp_hash_alg_t pgp_str_to_hash_alg(const char *);
 
+/* -----------------------------------------------------------------------------
+ * @brief   Returns output size of an digest algorithm
+ *
+ * @param   [in] alg
+ * @param   [out] output_length: must be not NULL
+ *
+ * @return  true if provided algorithm is supported and it's size was
+ *          correctly retrieved, otherwise false
+ *
+-------------------------------------------------------------------------------- */
+bool pgp_hash_digest_length(pgp_hash_alg_t alg, size_t *output_length);
```

Comment:
Aren't hash and digest are synonyms? imho pgp_hash_length or pop_digest_length would be enough.
Also from such function name it seems that it will return length, not put it in the variable. pop_get_hash_length seems to be more appropriate.